config/prow/plugins.yaml

```diff
@@ -340,6 +340,11 @@ blockades:
   blockregexps:
   - ^k8s.gcr.io/
   explanation: "k8s.gcr.io Image registry is frozen from April 2023. Please update the manifests in registry.k8s.io folder instead"
+- repos:
+  - kubernetes/cri-api
+  blockregexps:
```

Comment:
Should we copy the same exceptionregexps too?

Reply:
actually we should be removing that exception, recently https://github.com/kubernetes/client-go/pull/1394

Reply:
https://github.com/kubernetes/test-infra/pull/34210